exporters/sender/okhttp/src/main/java/io/opentelemetry/exporter/sender/okhttp/internal/OkHttpGrpcSender.java

```diff
@@ -214,7 +214,7 @@ private static String grpcMessage(Response response) {
   public CompletableResultCode shutdown() {
     client.dispatcher().cancelAll();
     if (managedExecutor) {
-      client.dispatcher().executorService().shutdownNow();
+      client.dispatcher().executorService().shutdown();
```

Comment:
do you know if the executor is using daemon threads, or do we need to worry about this possibly impacting application shutdown?